.github/workflows/maven.yaml

```diff
@@ -70,7 +70,7 @@ jobs:
         profile:
           - {name: 'unit-tests',    javaver: 11, args: 'verify -PskipQA'}
           - {name: 'qa-checks',     javaver: 11, args: 'verify javadoc:jar -Psec-bugs,errorprone'}
-          - {name: 'jdk17',         javaver: 17, args: 'verify javadoc:jar -Psec-bugs,errorprone'}
```

Comment:
was errorprone not working with jdk17?

Reply:
Also, I wonder if we should add a Java 21 build.

Reply:
Yeah errorprone was failing with jdk17 and I had no idea what to do to fix it.